manifests/reports.pp

```diff
@@ -19,6 +19,7 @@
   $dogapi_version,
   $hostname_extraction_regex = undef,
   $datadog_site = 'datadoghq.com',
+  $puppet_gem_provider = 'puppetserver_gem',
```

Comment:
Can we refer here to `$datadog_agent::params::gem_provider` instead?

Reply:
done